internal/service/dhcp/model_fixedaddress.go

```diff
@@ -0,0 +1,808 @@
+		MarkdownDescription: "The reference to the object.",
+	},
+	"agent_circuit_id": schema.StringAttribute{
+		Optional:            true,
+		Computed:            true,
+		MarkdownDescription: "The agent circuit ID for the fixed address.",
+	},
+	"agent_remote_id": schema.StringAttribute{
+		Optional:            true,
+		Computed:            true,
+		MarkdownDescription: "The agent remote ID for the fixed address.",
+	},
+	"allow_telnet": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "This field controls whether the credential is used for both the Telnet and SSH credentials. If set to False, the credential is used only for SSH.",
+	},
+	"always_update_dns": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "This field controls whether only the DHCP server is allowed to update DNS, regardless of the DHCP client requests.",
+	},
+	"bootfile": schema.StringAttribute{
+		Optional: true,
+		Computed: true,
+		Validators: []validator.String{
+			stringvalidator.AlsoRequires(path.MatchRoot("use_bootfile")),
+		},
+		MarkdownDescription: "The bootfile name for the fixed address. You can configure the DHCP server to support clients that use the boot file name option in their DHCPREQUEST messages.",
+	},
+	"bootserver": schema.StringAttribute{
+		Optional: true,
+		Computed: true,
+		Validators: []validator.String{
+			stringvalidator.AlsoRequires(path.MatchRoot("use_bootserver")),
+			customvalidator.IsValidFQDN(),
+		},
+		MarkdownDescription: "The bootserver address for the fixed address. You can specify the name and/or IP address of the boot server that the host needs to boot. The boot server IPv4 Address or name in FQDN format.",
+	},
+	"cli_credentials": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: FixedaddressCliCredentialsResourceSchemaAttributes,
+		},
+		Optional:            true,
+		Computed:            true,
+		MarkdownDescription: "The CLI credentials for the fixed address.",
+	},
+	"client_identifier_prepend_zero": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "This field controls whether there is a prepend for the dhcp-client-identifier of a fixed address.",
+	},
+	"cloud_info": schema.SingleNestedAttribute{
+		Attributes:          FixedaddressCloudInfoResourceSchemaAttributes,
+		Computed:            true,
+		MarkdownDescription: "Structure containing all cloud API related information for this object.",
+	},
+	"comment": schema.StringAttribute{
+		Optional: true,
+		Computed: true,
+		Validators: []validator.String{
+			stringvalidator.RegexMatches(
+				regexp.MustCompile(`^\S.*\S$`),
+				"Name should not have leading or trailing whitespace",
```

Comment:
```suggestion
				"Should not have leading or trailing whitespace",
```
Nitpick . As the error message isn't very clear !
Please check other fields that use this validation as well!

<img width="850" height="54" alt="image" src="https://github.com/user-attachments/assets/d493c935-c6f5-4fa4-9b36-92bc030543d8" />

Reply:
done